### PR TITLE
Prepare for 1.6.5 release and clean up the unused code.

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,13 @@ All Keras layers have been supported for conversion using keras2onnx since **ONN
 
 Windows Machine Learning (WinML) users can use [WinMLTools](https://docs.microsoft.com/en-us/windows/ai/windows-ml/convert-model-winmltools) to convert their Keras models to the ONNX format. If you want to use the keras2onnx converter, please refer to the [WinML Release Notes](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes) to identify the corresponding ONNX opset for your WinML version.
 
-keras2onnx has been tested on **Python 3.5, 3.6, and 3.7**, with **tensorflow 1.x** (CI build). It does not support **Python 2.x**.
+keras2onnx has been tested on **Python 3.5, 3.6, and 3.7**, with **tensorflow 1.x**/**tensorflow 2.0**//**tensorflow 2.1**  (CI build). It does not support **Python 2.x**.
 
 # Notes
+Keras2ONNX supports the new Keras subclassing model which was introduced in tensorflow 2.0 since the version 16.5. Some typical subclassing models like [huggingface/transformers](https://github.com/huggingface/transformers) have been converted into ONNX and validated by ONNXRuntime.<br>
```
Comment (jiafatom) [inline README.md]: version 16.5 -> version 1.6.5

File: README.md [hunk 2]
```diff
@@ -14,11 +14,13 @@ All Keras layers have been supported for conversion using keras2onnx since **ONN
 
 Windows Machine Learning (WinML) users can use [WinMLTools](https://docs.microsoft.com/en-us/windows/ai/windows-ml/convert-model-winmltools) to convert their Keras models to the ONNX format. If you want to use the keras2onnx converter, please refer to the [WinML Release Notes](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes) to identify the corresponding ONNX opset for your WinML version.
 
-keras2onnx has been tested on **Python 3.5, 3.6, and 3.7**, with **tensorflow 1.x** (CI build). It does not support **Python 2.x**.
+keras2onnx has been tested on **Python 3.5, 3.6, and 3.7**, with **tensorflow 1.x**/**tensorflow 2.0**//**tensorflow 2.1**  (CI build). It does not support **Python 2.x**.
```
Comment (jiafatom) [inline README.md]: // -> /

File: README.md [hunk 3]
```diff
@@ -127,9 +94,8 @@ pred_onnx = sess.run(None, feed)
 
 An alternative way to load onnx model to runtime session is to save the model first:
 ```
-import onnx
 temp_model_file = 'model.onnx'
-onnx.save_model(onnx_model, temp_model_file)
+keras2onnx.save_model(onnx_model, temp_model_file)
```
Comment (jiafatom) [inline README.md]: Why not using onnx.save_model?

Comment (wenbingl) [reply README.md]: save an import line.

**Pull request merged**